src/Sylius/Component/Core/spec/OrderProcessing/OrderShipmentProcessorSpec.php

```diff
@@ -96,6 +109,15 @@ function it_does_not_add_new_shipment_if_shipping_method_cannot_be_resolved(
 
         $order->isShippingRequired()->willReturn(true);
 
+        $productVariant1->isShippingRequired()->willReturn(true);
+        $productVariant2->isShippingRequired()->willReturn(true);
+
+        $itemUnit1->getShippable()->willReturn($productVariant1);
+        $itemUnit1->getShipment()->willReturn(null);
+
+        $itemUnit2->getShipment()->willReturn(null);
+        $itemUnit2->getShippable()->willReturn($productVariant2);
```

Comment:
Same code snippets written differently -- let's make them identical.

```suggestion
        $itemUnit1->getShippable()->willReturn($productVariant1);
        $itemUnit1->getShipment()->willReturn(null);

        $itemUnit2->getShippable()->willReturn($productVariant2);
        $itemUnit2->getShipment()->willReturn(null);

```